### ISSUE TITLE
python-mysqldb has gone away in Debian Bullseye

### ISSUE THREAD

**Issue by cjsoftuk** [PowerDNS/pdns-ansible#97]
Debian Bullseye removes many Python 2 libraries, leading to a failure to install python-mysqldb on a Bullseye box.  I'll try and build a suitable patch for this.

**Comment by cjsoftuk** [comment]
Hmm, so basically this involves removing the installation of python-mysqldb on Bullseye and newer.

What I don't see, though, yet, is a way to do this sensibly for just Bullseye and newer.

**Comment by absynth47215** [comment]
You can just remove the package from the Debian-specific code without issues. I just tried that and it works without problems on a freshly installed Debian 11 VM.

**Comment by sms-astanley** [comment]
I wanted to comment here before opening a new issue.  This also impacts Ubuntu 22.04.  Plays fail when enabling the MySQL backend due to `python-mysqldb` missing.
```
TASK [powerdns.pdns : Install the MySQL dependencies] **************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "No package matching 'python-mysqldb' is available"}
```
 Installations that don't use MySQL as a backend seem to work fine.

**Comment by Bodenhaltung** [comment]
Right, but we need also to be "backward compatible". This requires a little bit more effort (at least for testing).

**Comment by sms-astanley** [comment]
> Right, but we need also to be "backward compatible". This requires a little bit more effort (at least for testing).

Understood, I forked and removed the package just to verify.  Everything else works as expected.
